src/abstract-authenticated-command.js

```diff
@@ -8,27 +8,24 @@ class AbstractAuthenticatedCommand extends Command {
     if (!authenticator.getAuthToken()) {
       logger.info('Login required.');
       await authenticator.tryLogin({});
+      if(!authenticator.getAuthToken()) this.displayLoginMessageAndQuit();
```

Comment:
this is enough to stop when not logged. Adding a line here instead of wrapping the rest of the code.